### ISSUE TITLE
Non-interactive installation install only first plugin

### ISSUE THREAD

**Issue by zemanlx** [getgauge/gauge#751]
### Expected behavior
Non-interactive installation with list of plugins
```bash
$ sudo CI=true GAUGE_PLUGINS=java ./install.sh
Installed gauge binaries at /usr/local
Adding gauge to system path...
Gauge core successfully installed.

Installing plugin html-report ...
Installing plugin java ...
GAUGE_ROOT has been set in ~/.profile. If you face errors, run '$ source ~/.profile'
```

### Actual behavior
Only first plugin is installed (`html-report` in included by default)

```bash
$ sudo CI=true GAUGE_PLUGINS=java ./install.sh
Installed gauge binaries at /usr/local
Adding gauge to system path...
Gauge core successfully installed.

Installing plugin html-report ...
GAUGE_ROOT has been set in ~/.profile. If you face errors, run '$ source ~/.profile'
```

### Steps to reproduce
See behaviour

### Gauge version
```bash
$ gauge version
Gauge version: 0.9.0

Plugins
-------
html-report (4.0.1)
```

**Comment by sguptatw** [comment]
Test case
[CI-set up](https://travis-ci.org/sguptatw/docker/builds/260544377) to replicate the issue.

**Comment by BugDiver** [comment]
Should be fixed as part of  #752.
